README.md

```diff
@@ -296,6 +296,36 @@ noWeatherDamage=false
 ; The maximum range that you can place build objects at
 maximumPlacementDistance=5
 
+[Inventory]
```

Comment:
Worth mentioning this functionality further up in the `README.md` with the rest of the features!

Reply:
done, thanks